CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(Arbiter C CXX)
+
+cmake_minimum_required(VERSION 3.0)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 -pedantic -Wall -Wextra")
```

Comment:
Can you please re-add `-Wno-unused-parameter` here? It's for `library_folders`, because C doesn't permit unnamed parameters to designate lack of use.